docs/contributing/Style-guide-for-Flutter-repo.md

```diff
@@ -1522,12 +1168,7 @@ String capitalize(String s) {
 
 ### Use `=>` for inline callbacks that just return literals or switch expressions
 
-If your code is passing an inline closure that merely returns a list or
-map literal, or a switch expression, or is merely calling another function,
-then if the argument is on its own line, then rather than using braces and a
-return` statement, you can instead use the `=>` form. When doing this, the
-closing `]`, `}`, or `)` bracket will line up with the argument name, for
-named arguments, or the `(` of the argument list, for positional arguments.
+If your code is passing an inline closure that merely returns a list or map literal, or a switch expression, or is merely calling another function, or if the argument is on its own line, then rather than using braces and a return` statement, you can instead use the `=>` form. When doing this, the closing `]`, `}`, or `)` bracket will line up with the argument name, for named arguments, or the `(` of the argument list, for positional arguments.
```

Comment:
It looks like "a return\` statement" is missing the tick mark at the beginning of the word; could you add that in as well?